tests/test_launch_methods/test_jsrun/jobs.sh

```diff
@@ -0,0 +1,83 @@
+#!/bin/bash
+
+# Begin LSF Directives
+#BSUB -P BIP178
+#BSUB -W 0:30
+#BSUB -nnodes 2
+#BSUB -J jsrun_test
+#BSUB -o jsrun_test.%J
+#BSUB -e jsrun_test.%J
+#BSUB -N rct-tests@googlegroups.com
+
+# CONFIG
+path=`pwd`
+d=`date +%D | sed 's/\//_/g'`
+host=`/bin/hostname`
+#dest="rct-tests@googlegroups.com"
+dest="vivek.bala@rutgers.edu"
+
+run_cmd() {
+    res_set=$1
+    omp_env=$2
+    cmd=$3
+    exp_out=$4
+    tid=$5
+
+    # Create required folders
+    rm results_$d/$tid -rf
+    mkdir results_$d/$tid
+
+    # Record resource file, omp env, and cmd in log
+    echo $res_set >> results_$d/$tid/full_output.log
+    echo $omp_env >> results_$d/$tid/full_output.log
+    echo $cmd >> results_$d/$tid/full_output.log
+
+    # Set omp env, create resource file and run cmd
+    $omp_env
+    echo $res_set >> $path/resource-file
+    cmd=`echo $cmd | sed "s#resource-file#$path/resource-file#g"`
+    $cmd >> results_$d/$tid/jsrun_output.log
+    rm $path/resource-file
+
+    # Jsrun output to full output log file
+    echo "Output:" >> results_$d/$tid/full_output.log
+    cat results_$d/$tid/jsrun_output.log >> results_$d/$tid/full_output.log
+}
+
+assert() {
+    actual=$1
+    expected=$2
+    tid=$3
+
+    cp $actual results_$d/$tid/jsrun_output_dup.log
+    # Remove Node ID from actual output
+    sed -i 's/Node ....../Node /g' results_$d/$tid/jsrun_output_dup.log
```

Comment:
Why do you remove the IDs?

Reply:
Presumably because the differ from run to run, and are thus difficult to verify against a static output?

Reply:
Yes, that's correct. The node IDs differ between runs. The same can hold for HW threads in some tests.